view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js

```diff
@@ -137,7 +253,7 @@ define(
              * @return {boolean}
              */
             orderValueTooLow: function () {
-                return window.checkoutConfig.totalsData.base_grand_total < 3000;
+                return this.getTotal() < 3000;
```

Comment:
@jacstn wonder what is the idea behind this modification, why should we have an another function (`getTotal()`) to encapsulate this `window.checkoutConfig.totalsData.base_grand_total`

Reply:
we use it more than once. Was thinking that it is more readable than this long structure,

also we have stuff like:

```
            /**
             * Get Omise public key
             *
             * @return {string}
             */
            getPublicKey: function() {
                return window.checkoutConfig.payment.omise_cc.publicKey;
            },```

Reply:
ic